### ISSUE TITLE
Unsually high number of Rollback Transaction events in Legacy Processor

### ISSUE THREAD

**Issue by RiteshTC** [topcoder-platform/groups-api#22]
The Legacy Groups Processor has a lot of Transaction Rollback events.

    "code": "ER_DUP_ENTRY",
    "errno": 1062,
    "sqlState": "23000",
    "sqlMessage": "Duplicate entry '20000000-40983059-1' for key 'group_member_id'"

It may not be an issue, but the higher number of rollbacks on Authorization Db, might be impacting performance.
<img width="1093" alt="RollbackTransactions" src="https://user-images.githubusercontent.com/5771673/67070403-985e7f00-f19d-11e9-94b5-81e8bbee6547.png">

**Comment by ajefts** [comment]
@cwdcwd @dushyantb fyi, just making sure you guys are aware of this.

**Comment by bountyC0d3r** [comment]
@RiteshTC @ajefts yes, this is because we have not commented the code in `core-api` where it is also adding the members to the groups like `Wipro - All` and `Zurich`

Will get the code commented out and release in production.

**Comment by mtwomey** [comment]
If we comment out that code, will users no longer be added to those groups when they register?

**Comment by cwdcwd** [comment]
@mtwomey no as that logic is moved into the groups processor
